Bot/MessageHandlers/Commands/strike.js

```diff
@@ -0,0 +1,84 @@
+const ROLES = {
+    LUCKY : "902322660262957056"
+    STRIKEONE : "896026304237932565",
+    STRIKETWO : "906695092230832139",
+    BANNED : "614522888372092928",
+}
+
+function getMemberFromMention(mention, channel) {
```

Comment:
Should be in utils rather than the command. Command ideally should be kept very small just containing the actual "command" object